docs/index.md

```diff
@@ -1,9 +1,14 @@
+---
+layout: default
+title: Home
+nav_order: 1
+---
+
 # Teflo Webhooks Notification Plugin
+{: .no_toc }
+
 
 This is Teflo's plugin for sending teflo notifications to chat application using webhooks.
 This plugin is used to send notifications on triggers like `on_task`, `on_start`, `on_failure`, `on_demand`
 during and post teflo execution. For more information on how to setup triggers for notification
 please visit [here](https://teflo.readthedocs.io/en/latest/users/definitions/notifications.html#triggers).
```

Comment:
Could you also add the Teflo main page (the read the docs page ) , like you have added in other plugin docs

Reply:
yes, I'll create a PR for teflo repo to modify [this](https://teflo.readthedocs.io/en/latest/users/install.html) page in readthedocs

Reply:
No i dont mean to add anything on teflo, i wanted you to add the teflo's main page on this plugin's page

Reply:
done
https://github.com/RedHatQE/teflo/pull/109

Reply:
Ok, I'll add this line. my bad

Reply:
done